server_api.py

```diff
@@ -786,14 +786,14 @@ def edit_image_data(json_obj):
             "DO NOTHING"
         else :
             #get img_system_name
-            sql = ("SELECT img_system_name FROM image_data WHERE img_id=\"" + img_id + "\"")
-            pure_result = db.query(sql)
+            with ImageDao() as imageDao:
+                img_sys_name = imageDao.getImgSystemName(imgId=str(img_id))
             try: 
```

Comment:
I think the `try...except` block can be replace by `if img_sys_name...else`, because the queryOneValue() will return `None` while the SQL result is empty